man/memkind.3

```diff
@@ -986,6 +1044,11 @@ for parsing, so the syntax described in the
 .BR numa (3)
 man page for this routine applies: e.g. 1-3,5 is a valid setting.
 .TP
+.B MEMKIND_HBW_THRESHOLD
+This environment variable is bandwidth in MB/s that is the threshold for
+identifying high bandwidth memory. The default threshold is 204800 (200 GB/s),
+which is used if this environment is not set.
```

Comment:
"if this variable is not set" or maybe use the abbreviation "if this EV is not set"

Reply:
Done in #507 ( after merge #507 it will be available in this PR)